sdks/java/core/src/main/java/org/apache/beam/sdk/annotations/Experimental.java

```diff
@@ -101,5 +101,8 @@
 
     /** PCollection Schema support in Beam. */
     SCHEMAS,
+
+    /** {@code BeamSql}. */
+    SQL,
```

Comment:
My only hesitation is that everything else here is a part of the core SDK and SQL is "just a layer on top". It seems fine, but also it seems fine to just not have a `Kind`. Unless we use it to turn on/off experiments, which I don't think we do. Not sure the kind is ever used.

Reply:
I see. I was thinking if this is a right way to use Experimental annotation: add a kind and then add  annotations for that kind.

I will close this PR. We can reopen it if Kind is useful.